main/roller/roller.py

```diff
@@ -68,10 +86,30 @@ def roll_dice(dice_count, dice_sides, bonus):
         else:
             dice_sides = int(valid_roll.split('d')[1])
             dice_bonus = 0
-        rolled_dice = DiceRoller.roll_dice(dice_count, dice_sides, dice_bonus)
-        await ctx.send(
-            ROLLER_MESSAGES['success_result'].format(
-                rolled_dice['roll_results'], rolled_dice['bonus'],
-                rolled_dice['total']))
-    except AttributeError:  # Fail if valid_roll isn't created
+        if dice_sides >= 101:
```

Comment:
This is a bandaid. Will add correct error handling eventually (on to do list): https://trello.com/c/3EvzmT7j/12-add-error-handling